openedx_events/bridge/avro_attrs_bridge_extensions.py

```diff
@@ -56,13 +56,13 @@ def deserialize(self, data: str):
         return CourseKey.from_string(data)
 
     def record_fields(self):
-        """Define avro schema for self.cls."""
-        return AVRO_TYPE_FOR[str]
+        """Define Avro schema for self.cls."""
+        return PYTHON_TYPE_TO_AVRO_MAPPING[str]
 
 
 class DatetimeAvroAttrsBridgeExtension(AvroAttrsBridgeExtention):
     """
-    AvroAttrsBrdgeExtension for CourseKey class.
+    AvroAttrsBridgeExtension for CourseKey class.
```

Comment:
Can we drop a note about following the CloudEvents spec for timestamps: https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/avro-format.md#21-type-system-mapping?